src/rsz/src/RecoverPower.cc

```diff
@@ -223,10 +226,11 @@ void RecoverPower::recoverPower(const float recover_power_percent)
   if (resizer_->overMaxArea()) {
     logger_->error(RSZ, 125, "max utilization reached.");
   }
+  bad_vertices_.clear();
```

Comment:
On any exception this code won't be execute (eg "max utilization reached").

Reply:
Ok, didn't know that throws. I'll move it.

Reply:
Pushed the clear before the logs.